MapleServer2/Network/Session.cs

```diff
@@ -275,9 +276,12 @@ private void SendInternal(byte[] packet)
                 case SendOp.FIELD_ADD_USER:
                 case SendOp.FIELD_ENTRANCE:
                 case SendOp.SERVER_ENTER:
+                case SendOp.QUEST:
+                    break;
+                case SendOp.INTERACT_OBJECT:
+                    Logger.Debug($"SEND ({sendOp}): {packetString[Math.Min(packetString.Length, 6)..]}".Pastel("#4150B9"));
```

Comment:
Any reason to log this opcode in a different color? It's fine to do this for local debugging, but any reason this should be added to the repo as a whole?

Reply:
No. I was making it blue to debug it locally. I'll remove it.